mlflow/tracking/fluent.py

```diff
@@ -140,6 +142,11 @@ def start_run(run_id=None, experiment_id=None, run_name=None, nested=False):
                 "Cannot start run with ID {} because it is in the "
                 "deleted state.".format(existing_run_id)
             )
+        end_time = active_run_obj.info.end_time
+        _get_store().update_run_info(
+            existing_run_id, run_status=RunStatus.RUNNING, end_time=end_time
```

Comment:
Can you add a comment above this line explaining why we're setting `end_time` to the previous end_time (i.e. because this value is required, unfortunately)

Reply:
Done!